src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -6,13 +6,75 @@ import (
 	"io/ioutil"
 	"net"
 	"os"
+	"os/exec"
 	"path/filepath"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/moby/hyperkit/go"
 	"github.com/satori/go.uuid"
 )
 
+var defaultVPNKits = []string{"vpnkit",
```

Comment:
nit: can we newline for the individual entries and closing brace?  It'll make for cleaner diffs if we ever edit this